extensions/amp-story/1.0/amp-story-interactive-binary-poll.css

```diff
@@ -32,6 +32,8 @@
   height: 90px !important;
 	border-radius: 16px !important;
 	overflow: hidden !important;
+  /* overflow bug fix for percent bar animation on ios safari  */
+  -webkit-transform: translate3d(0, 0, 0) !important;
```

Comment:
I'm seeing that the compiler modifies prefixes and transforms in the build.
I'm looking into this now.